1.5/Defs/ThingDefs_Misc/FroggeMask.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Defs>
+    <ThingDef ParentName="HatMakeableBase">
+        <defName>Apparel_FroggeVisageMask</defName>
+        <label>frogge visage mask</label>
+        <description>A hard facial mask that resembles a stylized frogge face.</description>

```

Comment:
"mask of frogge. look like frogge. am frogge."?

Reply:
Changed Description